helm/prometheus-meta-operator/ci/default-values.yaml

```diff
@@ -9,6 +9,7 @@ Installation:
       Prometheus:
         Address: ""
         Host: ""
+        Letsencrypt: false
```

Comment:
I added that under Security.Letsencrypt.Enabled in my PR, do you want me to remove it?